filebeat/module/haproxy/log/_meta/fields.yml

```diff
@@ -32,10 +33,12 @@
     - name: captured_headers
       description: >
         List of headers captured in the request due to the presence of the "capture request header" statement in the frontend.
+      # Do we need text? does this even have be be indexed?
```

Comment:
Same as above. Yes, please index as keyword. It's an array of header names (no header values). Perfect fit for keyword.

Reply:
https://github.com/elastic/beats/pull/10397